Jenkinsfile_CNP

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "service"
-def ccdBranch = "master"
+def ccdBranch = "CIV-5708-second-PR"
```

Comment:
This is temporary for the purpose of testing this PR, I assume.

Don't forget to remove this change once you're finished.

Reply:
Thanks! Now removed and reverted to master